frontend/src/components/personal/notificationForm/NotificationForm.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { useFormik } from 'formik'
+import { Button, FormControl, FormGroup, FormControlLabel, Checkbox, Select, 
+  InputLabel, MenuItem, TextareaAutosize, Typography } from '@material-ui/core'
+
+import { successChangeUserNotification, 
+  userNotificationRequestFailed } from '../../../actions/notificationActions'
+import { RootState } from '../../../reducers/index'
+import { useStyles } from './NotificationForm.styles'
+import { FIVE } from '../../../constants/styles.values'
+import { getTimeDiffLocalTimeWithGMT, getTime } from '../../../utils/TimeCalculations'
+
+
+export default function PreferencesNotificationForm (): React.ReactElement {
+  const dispatch = useDispatch()
+  
+  const custom = useSelector((state: RootState) => state.notificationReducer)
+
+  const formik = useFormik({
+    enableReinitialize: true,
+    initialValues: {
+      sendEmailsWithNews: custom? custom.sendEmailsWithNews : true,
+      sendUpdatesThreads: custom? custom.sendUpdatesThreads: false,
+      sendUserReviews: custom? custom.sendUserReviews : false,
+      sendUserQuestsReviews: custom? custom.sendUserQuestsReviews : false,
+      sendUpdatesMessages: custom? custom.sendUpdatesMessages : false,
+      timezone: custom? custom.timezone :'UTC',
+      aboutUser: custom? custom.aboutUser : ''
+    },
+    onSubmit: (values) => {
+      console.log(values)
+        try {
+          dispatch(successChangeUserNotification(values))
```

Comment:
I think we should trigger an action that will initiate a request to backend, right?